notes/src/components/ListView/Note.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import './note.css';
+
+class Note extends React.Component {
+    constructor(props){
+        super()
+    }
+
+    
+    render() {
+      return (
+        <div className="note">
+          <Link to={`/note/${this.props.id}`} style={{textDecoration: 'none'}}>
+            <div className="noteTitle">{this.props.title}</div>
+          </Link>
+         <div className="noteText">{this.props.text.slice(0, 110)}...</div>
```

Comment:
As we talked about during the demo, you'll need something to check the length of a note or the lines of a note to have a max size and only do ellipses when they exceed that amount.